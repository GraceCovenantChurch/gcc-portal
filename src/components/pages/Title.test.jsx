import React from "react";
import { shallow } from "enzyme";
import Title from "./Title";
import renderer from "react-test-renderer";

describe("Title component", () => {
  let component, tree, wrapper;

  beforeEach(() => {
    component = renderer.create(<Title>RANDOM TEXT</Title>);
    tree = component.toJSON();

    wrapper = shallow(<Title />);
  });

  it("UNIT: should be defined", () => {
    expect(Title).toBeDefined();
  });

  it("SNAPSHOT: should render correctly", () => {
    expect(tree).toMatchSnapshot();
  });
});
