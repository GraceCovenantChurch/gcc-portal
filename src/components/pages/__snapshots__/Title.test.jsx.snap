// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component SNAPSHOT: should render correctly 1`] = `
<h2
  className="MuiTypography-root MuiTypography-h6 MuiTypography-colorPrimary MuiTypography-gutterBottom"
>
  RANDOM TEXT
</h2>
`;
